ionic/githubIonic/src/pages/login/login.ts

```diff
@@ -26,7 +27,8 @@ export class LoginPage {
         this.userService.signIn(this.username, this.password).subscribe(response => {
             localStorage.setItem('user-token', response.access_token);
             localStorage.setItem('refresh-token', response.refresh_token);
-            this.viewCtrl.dismiss();
+             let modal = this.modalCtrl.create(MyApp);
+              modal.present();
```

Comment:
I understand that, its pushing the whole app on top of the whole app. everytime they login a whole new app is created. also you're putting it in a modal so it's not the full app